Ubuntu CI broken outside of denoland/rusty_v8

GitHub Actions is configured to use a different base image for the main repository (`denoland/rusty_v8`) and for "other" repositories (think: personal forks).
https://github.com/denoland/rusty_v8/blob/5ca1964824fffe83198ca73f3db9b68fdb3cf2be/.github/workflows/ci.yml#L32

I'm not certain this was done because of GitHub Actions limits for personal accounts, or something else.

Linux CI fails with the following error:
```

Get:19 http://azure.archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [5792 B]
Get:20 http://azure.archive.ubuntu.com/ubuntu focal-backports/universe Translation-en [2060 B]
Get:21 http://azure.archive.ubuntu.com/ubuntu focal-backports/universe amd64 c-n-f Metadata [288 B]
Get:22 https://packages.microsoft.com/ubuntu/20.04/prod focal/main amd64 Packages [80.3 kB]
Get:23 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [745 kB]
Get:24 http://security.ubuntu.com/ubuntu focal-security/main Translation-en [148 kB]
Get:25 http://security.ubuntu.com/ubuntu focal-security/main amd64 c-n-f Metadata [8036 B]
Get:26 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [629 kB]
Get:27 http://security.ubuntu.com/ubuntu focal-security/universe Translation-en [96.2 kB]
Get:28 http://security.ubuntu.com/ubuntu focal-security/universe amd64 c-n-f Metadata [11.6 kB]
Fetched 4812 kB in 1s (3722 kB/s)
Reading package lists...
Building dependency tree...
Reading state information...
35 packages can be upgraded. Run 'apt list --upgradable' to see them.

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
Package gcc-5-aarch64-linux-gnu is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package g++-5-aarch64-linux-gnu
E: Unable to locate package g++-5-multilib
E: Package 'gcc-5-aarch64-linux-gnu' has no installation candidate
Error: Process completed with exit code 100.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -29,11 +29,11 @@ jobs:
             target: x86_64-apple-darwin
             variant: release
 
-          - os: ${{ github.repository == 'denoland/rusty_v8' && 'ubuntu-latest-xl' || 'ubuntu-latest' }}
+          - os: ${{ github.repository == 'denoland/rusty_v8' && 'ubuntu-latest-xl' || 'ubuntu-18.04' }}
             target: x86_64-unknown-linux-gnu
             variant: debug
 
-          - os: ${{ github.repository == 'denoland/rusty_v8' && 'ubuntu-latest-xl' || 'ubuntu-latest' }}
+          - os: ${{ github.repository == 'denoland/rusty_v8' && 'ubuntu-latest-xl' || 'ubuntu-18.04' }}
             target: x86_64-unknown-linux-gnu
             variant: release
 
@@ -41,11 +41,11 @@ jobs:
             target: x86_64-pc-windows-msvc
             variant: release # Note: we do not support windows debug builds.
 
-          - os: ${{ github.repository == 'denoland/rusty_v8' && 'ubuntu-latest-xl' || 'ubuntu-latest' }}
+          - os: ${{ github.repository == 'denoland/rusty_v8' && 'ubuntu-latest-xl' || 'ubuntu-18.04' }}
             target: aarch64-unknown-linux-gnu
             variant: debug
 
-          - os: ${{ github.repository == 'denoland/rusty_v8' && 'ubuntu-latest-xl' || 'ubuntu-latest' }}
+          - os: ${{ github.repository == 'denoland/rusty_v8' && 'ubuntu-latest-xl' || 'ubuntu-18.04' }}
             target: aarch64-unknown-linux-gnu
             variant: release
 

